src/VscPreviewExtension/src/tokenTreeCore.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { workspace, window, ExtensionContext, Uri } from "vscode";
+import * as childProcess from "child_process";
+import { TokenTreeContentProvider } from "./tokenTreeContentProvider";
+
+// Create a child process(DfmRender) by "_spawn" to render a html
+export class TokenTreeCore {
```

Comment:
Yes，it will be done in the next version.

---
In reply to: [77464922](https://github.com/dotnet/docfx/pull/664#discussion_r77464922) [](ancestors = 77464922)